build-core.sh

```diff
@@ -38,12 +37,23 @@ target_link_libraries (ripple_libs INTERFACE wasmedge)
 add_library (wasmedge::wasmedge ALIAS wasmedge)
 message(\"WasmEdge DONE\")
 " > Builds/CMake/deps/WasmEdge.cmake &&
+
+export LDFLAGS="-static-libstdc++"
+export CMAKE_EXE_LINKER_FLAGS="-static-libstdc++"
+export CMAKE_STATIC_LINKER_FLAGS="-static-libstdc++"
+
+git config --global --add safe.directory /io &&
 git checkout src/ripple/protocol/impl/BuildInfo.cpp &&
-sed -i s/\"0.0.0\"/\"$(date +%Y).$(date +%-m).$(date +%-d)-$(git rev-parse --abbrev-ref HEAD)+$4\"/g src/ripple/protocol/impl/BuildInfo.cpp &&
+sed -i s/\"0.0.0\"/\"$(date +%Y).$(date +%-m).$(date +%-d)-$(git rev-parse --abbrev-ref HEAD)$(if [ -n "$4" ]; then echo "+$4"; fi)\"/g src/ripple/protocol/impl/BuildInfo.cpp &&
+conan export external/snappy snappy/1.1.9@ &&
```

Comment:
See #546 
Our conanfile is using 1.1.10 
And the fact that we mispecified this again, is a really good argument for namespacing! 

```
conan export external/snappy snappy/1.1.10@xahaud/stable
conan export external/soci soci/4.0.3@xahaud/stable
```

I don't know about conan 2, but with conan 1, when we are exporting with only "soci/4.0.3@" and our recipe is different to rippled, which also exports using that name, you end up having to export again every time you switch between xahaud and rippled.

Also, as you can see, it's very easy for the builds to silently use the conan version.

Reply:
I'll fix this to 1.1.10.

and I'll fix the namespace once the PR is merged.